index.html

```diff
@@ -84,85 +84,40 @@ <h1 class="prof">I am Software Developer</h1>
-            <div class="project-container">
-                <img src="./image/pic.jpeg" alt="microverse">
-                <div class="desc">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <ul>
-                        <li>HTML5</li>
-                        <li>HTML5</li>
-                        <li>HTML5</li>
-                    </ul>
-                    <button type="button">See project</button>
-                </div>
-            </div>
-            <div class="project-container">
-                <img src="./image/pic.jpeg" alt="microverse">
-                <div class="desc">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <ul>
-                        <li>Ruby on Rails</li>
-                        <li>HTML5</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button">See project</button>
-                </div>
-            </div>
-            <div class="project-container">
-                <img src="./image/pic.jpeg" alt="microverse">
-                <div class="desc">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <ul>
-                        <li>Ruby on Rails</li>
-                        <li>HTML5</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button">See project</button>
-                </div>
-            </div>
-            <div class="project-container">
-                <img src="./image/pic.jpeg" alt="microverse">
-                <div class="desc">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <div class="list-lang"></div>
-                    <ul>
-                        <li>Ruby on Rails</li>
-                        <li>HTML5</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button">See project</button>
-                </div>
-            </div>
-            <div class="project-container">
-                <img src="./image/pic.jpeg" alt="microverse">
-                <div class="desc">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <div class="list-lang"></div>
-                    <ul>
-                        <li>Ruby on Rails</li>
-                        <li>HTML5</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <button type="button">See project</button>
+        <div class="projects-desktop"></div>
+
+        <div id="modal-wrapper" class="modal-invisible">
+            <!-- <div id="modal">
+                <div id="modal-snapshot">
+                    <div id="modal-cancel">
+                        <i onclick=closeModal() class="fa fa-times" title="medium" aria-hidden="true"></i>
+                    </div>
+                    <img id="screenshot" src="./image/pic.jpeg" alt="project snapshot">
```

Comment:
- [ ] Please avoid keeping commented code in your codebase because it does not look clean.